pkg/services/config.go

```diff
@@ -778,11 +809,22 @@ func New(ctx context.Context, config Config) (*Services, error) {
 			CodeChallengeMethodsSupported:     []string{"S256", "plain"},
 			TokenEndpointAuthMethodsSupported: []string{"client_secret_basic", "client_secret_post", "none"},
 		},
-		AccessControlRuleHelper: acrHelper,
-		WebhookHelper:           mcp.NewWebhookHelper(mcpWebhookValidationInformer.GetIndexer()),
-		LocalK8sConfig:          localK8sConfig,
-		MCPServerNamespace:      config.MCPNamespace,
-		K8sSettingsFromHelm:     helmK8sSettings,
+		AccessControlRuleHelper:   acrHelper,
+		WebhookHelper:             mcp.NewWebhookHelper(mcpWebhookValidationInformer.GetIndexer()),
+		LocalK8sConfig:            localK8sConfig,
+		MCPServerNamespace:        config.MCPNamespace,
+		K8sSettingsFromHelm:       helmK8sSettings,
+		ServerUpdateCheckInterval: time.Duration(config.UpdateCheckIntervalMins) * time.Minute,
+		Distribution: func() string {
```

Comment:
I think you got confused on distribution here.

The engine is meant to be kubernetes vs docker (vs local) and elsewhere you properly pass MCPRuntimeBackend as the engine value.

Distribution is meant to be oss vs enterprise.

...sorry, probably a bit more work to figure that out.

Reply:
Fixed